Sources/HummingbirdWSCore/WebSocketInboundStream.swift

```diff
@@ -78,7 +78,7 @@ public final class WebSocketInboundStream: AsyncSequence, Sendable {
                         return .init(from: frame)
                     default:
                         // if we receive a reserved opcode we should fail the connection
-                        self.handler.logger.trace("Received reserved opcode", metadata: ["opcode": .stringConvertible(frame.opcode)])
+                        self.handler.logger.debug("Received reserved opcode", metadata: ["hb.opcode": .stringConvertible(frame.opcode)])
```

Comment:
Can we namespace this to websocket?

Reply:
`hb.ws.opcode`?

Reply:
or `[WebSocket] Received reserved opcode`?

Reply:
`hb.ws.opcode` is what I was thinking

Reply:
done